public/styles.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
```

Comment:
Should this be included, as it seems it's coming from another MR?

Reply:
Branched off from branding branch to test stylelint for css files